packages/runtime/src/extensibility/facades/transport/RelationshipsFacade.ts

```diff
@@ -68,6 +80,22 @@ export class RelationshipsFacade {
         return await this.terminateRelationshipUseCase.execute(request);
     }
 
+    public async reactivateRelationship(request: ReactivateRelationshipRequest): Promise<Result<RelationshipDTO>> {
```

Comment:
What do you think about renaming this (and the use case/underlying RelationshipsController function) to `initiateRelationshipReactivation`? This would yield a nice consistency with the other names and is in accordance to what we did for the Identity deletion.

Reply:
I agree with renaming, would personally tend to `requestRelationshipReactivation` and using it everywhere (the audit log entra reason is also called like this)

Reply:
This would then yield a `RequestRelationshipReactivationRequest`. For this reason and for symmetry with Identity deletion I'd prefere `initiate...`. Having said that, there are also other places where our naming is not super pretty. So, if you feel strongly about it, I can also be convinced of another name.

Reply:
Identity deletion doesn't require your peer to approve, while requesting the reactivation is a request in everything but technical realization.

Reply:
I see your point. `requestRelationshipReactivation` is fine for me. 👍